tools/Google.Cloud.Tools.ReleaseManager/ContainerCommands/PackageLibraryCommand.cs

```diff
@@ -42,9 +48,105 @@ public int Execute(ContainerOptions options)
         }
         Processes.RunBashScript(Path.Combine(repoRoot, "docs"), "builddocs.sh", apis.Select(api => api.Id));
 
-        // TODO: We need to create the relevant docs bundles to push later. Ideally, we'll create tgz files
-        // ready to just upload. We can probably call into DocUploader directly.
-
+        foreach (var api in apis)
+        {
+            PackageDocumentation(rootLayout, api, outputDirectory);
+        }
         return 0;
     }
+
+    // This is the container equivalent of running uploaddocs.sh, but creating a tgz file instead of
+    // directly pushing to GCS. The code for metadata and compression is taken from DocUploader, but we may be
+    // able to retire that tool later.
+    private static void PackageDocumentation(RootLayout rootLayout, ApiMetadata api, string outputDirectory)
+    {
+        var docsDir = rootLayout.CreateDocsLayout(api.Id).OutputDirectory;
```

Comment:
For future refactoring, maybe this returns an object DocsLayout, that has all the properties SiteDir, DevSiteDir, etc.

Reply:
DocsLayout already exists - we just don't have the extra output directories as properties :)